### PR TITLE
1398: Add release notes for v2.7.0

### PR DESCRIPTION
**Pull request by benedeki**

### CODE REVIEW

File: _posts/2020-06-12-release-2.7.0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+excerpt_separator: <!--more-->
+---
+
+As of 12/06 2020 the new version 2.7.0 is out
+<!--more-->
+
+## Standardization & Menas Improvements
+
+- [#1235]({{ site.github.issues_url }}/1235) [BinaryType]({{ site.baseurl }}/docs/2.0.0/usage/schema#binary) is now supported 
+both in Standardization/Conformance and in menas UI. Mind the [`encoding`]({{ site.baseurl }}/docs/2.0.0/usage/schema#encoding)
```
Comment (Zejnilovic) [inline _posts/2020-06-12-release-2.7.0.md:14]:
```suggestion
both in Standardization/Conformance and in Menass UI. Mind the [encoding]({{ site.baseurl }}/docs/2.0.0/usage/schema#encoding) 
```
The encoding in backquotes will not be as a link by anyone. I would have it as a plain text.

Link in backquotes:
image

Comment (benedeki) [reply _posts/2020-06-12-release-2.7.0.md:14]: I dunno. I find the formatting more important then noticing the link. After all the BinaryType as a whole is linked visibly. This is just something for the observant ones going directly. Others' opinons?

**Comment by benedeki** [comment]
> 
> 
> Also please mind that the file name of posts in `_posts` is a date of publishing.
> 
> Now there a few ways we can go:
> 
>     * have it always as the date of releasing to maven
>       
>       * it might get overshadowed by some news or articles that we post there. If we take a too long time, the release notes won't be the newest thing in the news section.
>       * user will be able to find a release note based on the release date
> 
>     * have it as a date of actually publishing
>       
>       * no race conditions of articles
>       * we could implement proper date search or put a date of release in tags in a "nice" format
> 
>     * generate the release notes first thing on the date of the release and ask everyone to spare 5 minutes to check them and have them out that same day. Best solution in my head.

You are right. I will make better effort to combine 1 and 3 in the future releases.

File: _posts/2020-06-12-release-2.7.0.md (hunk 1 above)
Comment (dk1844) [reply _posts/2020-06-12-release-2.7.0.md:14]: @Zejnilovic: `Menass` -> `Menas`?

Comment (Zejnilovic) [reply _posts/2020-06-12-release-2.7.0.md]: > @Zejnilovic: `Menass` -> `Menas`?

I might have made a mistake there

Comment (benedeki) [reply _posts/2020-06-12-release-2.7.0.md]: In any case it's Menas now, thanks

**Pull request merged**